src/DB/projects.json

```
@@ -2202,5 +2202,23 @@
         "tech": ["python","Machine learning"]
       }
```

just paste the sample code to here 
```
{
        "link": "https://github.com/Abhinavcode13/Game-Of-Thrones-Personality-Matcher",
        "title": "Game of thrones personality matcher",
        "description": "This is a project based on data science and machine learning which used TSNE algorithm and the main aim of this project is to match the personality of the game of thrones characters and this app is hosted on streamlit framework in pycharm IDE",
        "tech": ["python","Machine learning","Jupyter Notebook"]
      }
```

```suggestion
      },
      {
        "link": "https://github.com/Abhinavcode13/Game-Of-Thrones-Personality-Matcher",
        "title": "Game of thrones personality matcher",
        "description": "This is a project based on data science and machine learning which used TSNE algorithm and the main aim of this project is to match the personality of the game of thrones characters and this app is hosted on streamlit framework in pycharm IDE",
        "tech": ["python","Machine learning","Jupyter Notebook"]
      }
```